demos/tooltip.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2016 Google Inc. All rights reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      https://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License
+  -->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>MDL Tooltip Demo</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <style>
+      /* Make the demo look a little bit better */
+    </style>
+    <script src="assets/material-design-lite.css.js" charset="utf-8"></script>
+  </head>
+  <body>
+    <main>
+      <h1>MDL Tooltip</h1>
+      <i class="material-icons mdl-tooltip" aria-label="Favorite">favorite</i>
+      <i class="material-icons mdl-tooltip" aria-label="Plus One">plus_one</i>
+
+      <button class="mdl-button mdl-tooltip" aria-label="Complete">
```

Comment:
eh, kind of a weird use case right? Maybe if the button were an icon button it would work...perhaps we should put a TODO in here saying this should be an icon button?